pytest_allure_dsl.py

```diff
@@ -12,7 +12,7 @@
 except ImportError:
     MarkInfo = None
 from allure.constants import Label, AttachmentType
-from allure.pytest_plugin import MASTER_HELPER, LazyInitStepContext
+from allure.pytest_plugin import MASTER_HELPER as allure, LazyInitStepContext
```

Comment:
If rewrite in a way 
`allure = MASTER_HELPER`
then it will be a correct suggestion from IDE(PyCharm) to import exactly this allure.